linkerd/app/inbound/src/http/mod.rs

```diff
@@ -238,3 +238,127 @@ fn trace_labels() -> std::collections::HashMap<String, String> {
     l.insert("direction".to_string(), "inbound".to_string());
     l
 }
+
+#[cfg(fuzzing)]
+pub mod fuzz_logic {
+    use crate::{
+        target::{HttpAccept, TcpAccept, TcpEndpoint},
+        test_util::{
+            support::{connect::Connect, http_util, profile, resolver},
+            *,
+        },
+        Config, Inbound,
+    };
+    use hyper::http;
+    use hyper::{client::conn::Builder as ClientBuilder, Body, Request, Response};
+    use linkerd_app_core::{
+        io::{self, BoxedIo},
+        proxy,
+        svc::{self, NewService, Param},
+        tls,
+        transport::{ClientAddr, Remote, ServerAddr},
+        Conditional, Error, NameAddr, ProxyRuntime,
+    };
+    pub use linkerd_app_test as support;
+    use linkerd_app_test::*;
+
+    pub async fn fuzz_entry_raw(uri: &str, header_data_1: &str, header_data_2: &str, is_get: bool) {
```

Comment:
Hmm, it looks like `header_data_1` is actually the header _name_, while `header_data_2` is the _header value_. What if we called these
```suggestion
    pub async fn fuzz_entry_raw(uri: &str, header_name: &str, header_value: &str, is_get: bool) {
```